### PR TITLE
respects uid content type to generate xref properties

### PR DESCRIPTION
**Pull request by OsmondJiang**
fix: https://ceapex.visualstudio.com/_workitems/edit/110999?fullScreen=True and https://ceapex.visualstudio.com/_workitems/edit/111001?fullScreen=True

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/docfx/pull/4962)

### CODE REVIEW

**Comment by OsmondJiang** [comment]
check this: 
image

**Comment by OsmondJiang** [comment]
Diff are expected, take DevSandbox as example, the v2 also throw uid not found errors:

image

File: src/docfx/lib/schema/JsonSchemaTransformer.cs [hunk 1]
```diff
@@ -62,6 +67,7 @@ List<InternalXrefSpec> LoadXrefSpecs(JsonSchema schema, JToken node)
 
                         break;
                     case JArray array:
+
```
Comment (mingwli) [inline src/docfx/lib/schema/JsonSchemaTransformer.cs]: an empty line

Comment (live1206) [inline src/docfx/lib/schema/JsonSchemaTransformer.cs]: remove blank line

Comment (OsmondJiang) [reply src/docfx/lib/schema/JsonSchemaTransformer.cs]: we leave a blank line under `case` habitually

Comment (live1206) [reply src/docfx/lib/schema/JsonSchemaTransformer.cs]: do we? I barely see that

Comment (OsmondJiang) [reply src/docfx/lib/schema/JsonSchemaTransformer.cs]: removed, follow the existing style.

**Pull request merged**